networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/P2PConfig.java

```diff
@@ -55,9 +55,9 @@ public class P2PConfig {
   // DEFAULT_BACKGROUND_THREAD_COUNT)
   // The storage query channel allows up to 10 parallel queries (STORAGE_QUERY_CHANNEL_PARALLELISM)
   // To avoid resource saturation and ensure capacity for other tasks, we limit historical data
-  // queries to 5
-  public static final int DEFAULT_HISTORICAL_DATA_MAX_CONCURRENT_QUERIES = 5;
-  public static final int DEFAULT_HISTORICAL_MAX_QUERY_QUEUE_SIZE = 100_000;
+  // queries to 3
+  public static final int DEFAULT_HISTORICAL_DATA_MAX_CONCURRENT_QUERIES = 3;
+  public static final int DEFAULT_HISTORICAL_MAX_QUERY_QUEUE_SIZE = 5_000;
```

Comment:
I'd test it somewhere. I see sepolia-archive is the most RPC active one currently

Reply:
everything is flat to 0 there.
<img width="1051" height="421" alt="image" src="https://github.com/user-attachments/assets/e9894e10-18ee-4799-8288-547a3895cea4" />

All mainnets are flat at 0 too but Holesky boxes are interesting. Going to deploy this there.